docs/reference/sql/functions/math.asciidoc

```diff
@@ -385,7 +385,7 @@ include-tagged::{sql-specs}/docs/docs.csv-spec[mathInlineSqrt]
 --------------------------------------------------
 
 [[sql-functions-math-truncate]]
-==== `TRUNCATE`
+==== `TRUNCATE/TRUNC`
```

Comment:
Shouldn't be `TRUNC/TRUNCATE` (alphabetical order)?

Reply:
In general, we don't do it alphabetically here, for example: `DAY_OF_MONTH/DOM/DAY`
@astefan, what do you think?

Reply:
Seems indeed a mixed usage: `CEIL/CEILING`, but also `RANDOM/RAND`.
Fwiw, I find the "short/long" version easier to read (i.e. visually take in), but unless we order them all like that, this one case is not very relevant.

Reply:
I was thinking also that TRUNCATE was already there, if we put TRUNC first we will need to also change the definition paragraph and use TRUNC instead (we always use the 1st version there).

Reply:
The initial implementation for RANDOM and TRUNCATE came from https://docs.microsoft.com/en-us/sql/odbc/reference/appendixes/numeric-functions?view=sql-server-2017, where they are indeed called like that. I agree with Marios here: in our docs the first name is the "main" one and the rest are aliases. For consistency, I would go with TRUNCATE/TRUNC, RANDOM/RAND etc. The biggest counter-example to the "alphabetical" approach is the series of DOM/DOY/DOW/DAYNAME aliases for some of the date/datetime functions.